### PR TITLE
css-three-column-layout Timothy Lee

### PR DESCRIPTION
**Pull request by timlee712**
css-three-column-layout

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <div class="column-third">
+          <img src="images/cloud-computing.png" alt="cloud" id="cloud">
+        </div>
+        <div class="column-third">
+          <img src="images/discussion.png" alt="discussion" class="discussion">
+        </div>
+      </div>
+      <div class="row">
+        <div class="text">
+          <h3>Integrates with existing tools</h3>
```
Comment (JenBowman1) [inline css-three-column-layout/index.html:30]: The text content for this element should match the reference image

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
Just a couple small changes! Please make sure that the text for the headers do not wrap onto multiple lines. If you look at the reference image, they are all on one line

**Comment by timlee712** [comment]
css-three-column-layout

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**